.github/workflows/reusable_build.yml

```diff
@@ -34,9 +38,11 @@ on:
         default: ''
         type: string
       run_for_devices:
-        description: 'The list of device(s) on which the test will run (defaults to ["nanos", "nanox", "nanosp", "stax"])'
+        description: |
+          The list of device(s) on which the test will run. Defaults to ["nanos", "nanox", "nanosp",
+          "stax"], except if the application is configured with a 'ledger_app.toml' file at its root.
```

Comment:
Confusing, reading this lets me think that the manifest has precedence over the param

Reply:
Which is not the case, so I need to change the phrasing.

Reply:
I recycled the phrasing you proposed for the `_get_app_metadata.yml:compatible_devices` which was clearer.